libnavigation-ui/src/test/java/com/mapbox/navigation/ui/map/MapFpsDelegateTest.java

```diff
@@ -60,7 +75,7 @@ public void adjustFpsFor_thresholdSetWithCorrectManeuver() {
     when(batteryMonitor.isPluggedIn(any(Context.class))).thenReturn(false);
     MapFpsDelegate delegate = new MapFpsDelegate(mapView, batteryMonitor);
     RouteProgress routeProgress = buildRouteProgressWith("straight");
-    int maxFps = 5;
+    int maxFps = 30;
```

Comment:
Aren't we changing the test scenario here? Why was this needed `5` > `30`? Does the test fail with `5`?

Reply:
Made the change locally to test something but forgot to revert it back. Thanks for flagging. Fixed.